charms/kfp-api/metadata.yaml

```diff
@@ -4,6 +4,7 @@ description: |
   Machine learning (ML) toolkit that is dedicated to making deployments
   of ML workflows on Kubernetes simple, portable, and scalable.
 series: [kubernetes]
+min-juju-version: "2.8.0"
```

Comment:
Why are these ones `2.8.0` and in other PRs its `2.9.0`?

Reply:
it's an escapee. let me fix that